R-package/R/lgb.Dataset.R

```diff
@@ -710,7 +710,9 @@ Dataset <- R6::R6Class(
 #' @title Construct \code{lgb.Dataset} object
 #' @description Construct \code{lgb.Dataset} object from dense matrix, sparse matrix
 #'              or local file (that was created previously by saving an \code{lgb.Dataset}).
-#' @param data a \code{matrix} object, a \code{dgCMatrix} object or a character representing a filename
+#' @param data a \code{matrix} object, a \code{dgCMatrix} object or
```

Comment:
One `or` is already used for a binary file:
> or a LightGBM Dataset binary file
```suggestion
#' @param data a \code{matrix} object, a \code{dgCMatrix} object,
```